packages/hash/frontend/src/pages/shared/auth-modal-layout.tsx

```diff
@@ -16,7 +16,6 @@ export const AuthModalLayout: FunctionComponent<AuthModalLayoutProps> = ({
   loading,
 }) => (
   <Dialog
-    as="div"
     open={show}
     onClose={onClose ?? (() => {})}
     style={{
```

Comment:
Do we need all these styles or can we remove at least some?

Reply:
Did not wanted to modify this component further, since it's unused right now and we might possibly remove it.
[Related slack thread](https://hashintel.slack.com/archives/C022217GAHF/p1670931004428459) _(internal)_